transcom/mymove
# INT-B-21456-shipment-email-modification-for-retiree-separatee-counselor-approved-move-details

**NOTE: Here is a doc to the the screenshots** [B-21456.docx](https://github.com/user-attachments/files/17368157/B-21456.docx)
## [Agility [B-21456]](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1015562&RoomContext=TeamRoom%3A1007470&concept=TeamRoom)

## Summary
Currently when a  Counselor submits a move details , emails that gets sent to retirees and separatees state that their move is scheduled to deliver to their HOS/HOR/PLEAD instead of the delivery address on their shipment. So these changes are to let the delivery address on the email to display the right address.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

**Local testing for emails, you will need to start the server with this:**
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run

### How to test

1. Access the application
2. Login as a Customer
3. Create a new move (Select Retirement/ Separation) on the "Tell us about your move orders page".
4. Go through the process of Setting  up a shipments (e.g HHG/PPM) to create a shipment , sign a complete(Make sure to copy/save your move code for later). 
5. Login as a Service Counselor(SC)
6. Search for your move code in step 4 and navigate to it.
7. Make sure you edit/fix any required fields.
8. Click on "Submit move details" and click "Yes, submit" button on the modal.
9. You will receive email with subject"YOur counselor has approved your move details" in the email you provided during the Move/shipment creation process.
10. Note that the verbiage in the email "This is a confirmation that your counselor has approved move details for the assigned move code [YOUR_MOVE_CODE] to [**Destination_address_you_provided_for_shipment**].

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![image](https://github.com/user-attachments/assets/3dc352bb-b809-46ee-84f8-dea982c36ab1)
![image](https://github.com/user-attachments/assets/4fad0a1e-1f60-4f98-96ef-aca27d0fc0d6)
![image](https://github.com/user-attachments/assets/d67243f7-d6e8-40dc-ba4f-8ff465afef4d)
![image](https://github.com/user-attachments/assets/1e96c343-c926-472d-aa4b-5fb26d031c5d)
![image](https://github.com/user-attachments/assets/7dffde16-ff96-4f02-b226-50ed52bdcefa)
![image](https://github.com/user-attachments/assets/d656f512-38f3-4652-9187-556c98a864b3)
